maven-build-scan-capture-extension/src/main/java/com/gradle/MavenBuildScanCaptureListener.java

```diff
@@ -56,16 +60,14 @@ public void configure(GradleEnterpriseApi gradleEnterpriseApi, MavenSession sess
         });
 
         // Capture build scan link
-        gradleEnterpriseApi.getBuildScan().buildScanPublished(buildScan -> {
-            captureBuildScanLink(buildScan.getBuildScanUri().toString());
-        });
+        gradleEnterpriseApi.getBuildScan().buildScanPublished(this::captureBuildScanLink);
```

Comment:
ah, we're going to have to come up with a backward compatibility strategy for this action.

Reply:
I will postpone merging this PR in order to directly integrate it with the Develocity API.
There is no need to rush a version with the GE API

Reply:
Sounds good.